### DISCUSSION TITLE
DJANGO_CELERY_BEAT_TZ_AWARE Not effective

### DISCUSSION THREAD

**Discussion by xrhh** [celery/django-celery-beat#488] [Q&A]
### Summary:

Include a *brief* descrioption of the problem here, and fill out the version info below.  

* Celery Version: 5.1.2
* Celery-Beat Version: 2.2.1

my setting.py:
TIME_ZONE = 'Asia/Shanghai'
USE_TZ = False
CELERY_TIMEZONE = 'Asia/Shanghai'
CELERY_ENABLE_UTC = False
DJANGO_CELERY_BEAT_TZ_AWARE = False

### Detailed information
I set DJANGO_CELERY_BEAT_TZ_AWARE = False,
Crontab Schedule is correct,
But for Clocked Schedule the time zone is still wrong，The decifference is 8 hours

**Comment by auvipy** [comment]
why not try True

**Comment by xrhh** [comment]
> why not try True

I tried, but it didn't work

**Comment by 68110923** [comment]
timezone_aware = False

**Comment by nick2wang** [comment]
exactly the same issue as yours, did u solve it out?
